fgpyo/util/metric.py

```diff
@@ -464,6 +465,7 @@ def __init__(
             exclude_fields: If specified, any listed fieldnames will be excluded when writing
                 records to file.
                 May not be used together with `include_fields`.
+            kwargs: any keyword arguments to be passed to csv.DictWriter.
```

Comment:
Since we already bass in `f`, `fieldnames`, and `delimiter`.  Should we (a) document that those should not be given, and (b) throw an exception if they're present in `kwargs`?